examples/protocol_specification_ex/sample.yaml

```diff
@@ -9,6 +9,8 @@ protocol_specification_id: fetchai/two_party_negotiation:0.1.0
 speech_acts:
```

Comment:
Revert changes here to keep it simple. This is only an example and doesn't have to have different type cases in it. We have the test specs and protocols for all possibilities.

Reply:
done